src/fides/api/ctl/migrations/versions/de456534dbda_add_privacyrequest_consent_preferences_.py

```diff
@@ -0,0 +1,43 @@
+"""add privacyrequest consent preferences and ruleuse table
+
+Revision ID: de456534dbda
+Revises: 3caf11127442
+Create Date: 2023-01-03 22:59:45.144538
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "de456534dbda"
+down_revision = "3caf11127442"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "privacyrequest",
+        sa.Column(
+            "executable_consent_preferences",
+            postgresql.JSONB(astext_type=sa.Text()),
+            nullable=True,
+        ),
+    )
+
+    op.add_column(
+        "consentrequest", sa.Column("privacy_request_id", sa.String(), nullable=True)
```

Comment:
If a privacy request is created to propagate consent request preferences, we link it on the ConsentRequest for record-keeping.  A privacy request may not always be created, for example, if none of the consent preferences ended up being executable, or something failed with privacy request creation.